map/map_projection_loader/test/data/projection_info_local.yaml

```diff
@@ -0,0 +1,6 @@
+type: MGRS
```

Comment:
```suggestion
type: Local
```

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/3986/commits/b358ab93eb297997ec416269b7738fac822a4733